shared/macros/10-oval.jinja

```diff
@@ -663,7 +663,7 @@ Generates the :code:`<affected>` tag for OVAL check using correct product platfo
   </linux:rpminfo_object>
   {{% if evr %}}
     <linux:rpminfo_state id="ste_{{{ test_id }}}" version="1">
-      <linux:evr datatype="evr_string" operation="greater than or equal">{{{ evr }}}</linux:evr>
+      <linux:evr datatype="evr_string" operation="{{{ evr_op }}}">{{{ evr }}}</linux:evr>
```

Comment:
How will the equality operation work with RPM packages? For example, I want to check `firefox`. I have `firefox-107.0-4.fc37.x86_64` installed. I have `platform: package[firefox]==107.0-4` in rule.yml. The result is notapplicable. I can't define `platform: package[firefox]==107.0-4.fc37` because that causes a build error. The collected items in ARF contain `<lin-sys:evr datatype="evr_string">0:107.0-4.fc37</lin-sys:evr>`. It seems to me that I can't have `==` operator for RPM packages in platform definitions. How does that work?

Reply:
This is also the limitation of distutils as it does not support extra letters in versions. We can work around with `package[firefox]>=107.0-4,<107.0-5` if we absolutely have to.

Reply:
But in general we might need to think about it and amend OVAL/OpenSCAP in some way because `package[firefox]==107.0-4.fc37` makes platform distro-version specific, which we kinda trying to get rid of.